coala_quickstart/generation/FileGlobs.py

```diff
@@ -60,3 +60,63 @@ def get_project_files(log_printer, printer, project_dir):
         ignored_file_paths=ignore_path_globs)
 
     return file_paths
+
+
+def generate_glob_exps(project_files, extensions, project_dir):
+    """
+    Generate glob expressions that matches exactly the files
+    with the given extensions in the user's project directory.
+
+    :param project_files:
+        The list of files to match.
+    :param extensions:
+        A list of file extensions.
+    :param project_dir:
+        The user's project directory.
+    :return:
+        A tuple with the first element (a list of glob expressions)
+        representing files to include and the second element (a list
+        of glob expressions) representing the files to exclude.
+    """
+    globs = []
+    extlist = set()
+    included_globs = []
+    excluded_globs = []
+
+    for root, dir, files in os.walk(project_dir):
```

Comment:
@hypothesist 
A. You aren't using `dir` anywhere in this block. Maybe use `_` instead?
B. If you still wanna keep some name then maybe use something that's not a keyword? `dir` is an inbuilt function afterall.